catalog/geoscience-aus-cat/catalog.jsonnet

```diff
@@ -0,0 +1,30 @@
+local id = 'geoscience-aus-cat';
```

Comment:
Because we have gee:skip_indexing set, this file shouldn't exist yet (which is annoying, and we will simplify this process). Please remove it from the PR, and I will readd it when the datasets are activated.